src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -36,13 +37,43 @@ module.exports = class LevelPlane extends Array {
   setBlockAt(position, block) {
     this[this.coordinatesToIndex(position)] = block;
 
+    if (block.isRedstone) {
+      this.determineRedstoneSprite(position);
+      this.getOrthogonalPositions(position).forEach(orthogonalPosition => {
+        const orthogonalBlock = this.getBlockAt(orthogonalPosition);
+        if (orthogonalBlock && orthogonalBlock.isRedstone) {
+          this.determineRedstoneSprite(orthogonalPosition);
+          //this.levelModel.controller.levelView.refreshActionPlane(this.levelModel, [this.coordinatesToIndex(orthogonalPosition)]);
```

Comment:
Commented out?  If we do the refresh in this method we shouldn't need to refresh from the Controller.  But this would only refresh the touching blocks, not the placed (or destroyed) block.